linear_operator/operators/cat_linear_operator.py

```diff
@@ -244,6 +244,14 @@ def _getitem(
                     res_list.append(res)
 
         elif torch.is_tensor(cat_dim_indices):
+            if cat_dim_indices.dim() > 1:
+                # Supporting broadcasting tensor indices for CatLinearOpeartor is very challenging to implement,
+                # and probvably not a featuer we need.
```

Comment:
provably not a future we need? 

```suggestion
                # and probably not a feature we need.
```